user/reference/bionic.md

```diff
@@ -104,11 +104,20 @@ The following versions of Docker, version control software and compilers are pre
 
 ## Python support
 
-* Supported Python versions: `2.7`, `3.6` or higher.
-* Pre-installed Python versions: `2.7.15`, `3.6.7`, and `3.7.1`.
-* Python `2.7.15` will be used when no language version is explicitly set.
+Images for all languages have Python 2 and 3 installations provided by the distribution. The Python 2 one has pip preinstalled.
```

Comment:
Given that Python 2 end of life is in 57 days does, in make sense to:
~The Python 2 one has pip preinstalled.~
__The Python 3 one has pip preinstalled.__

Reply:
With sections, the text looks very spread out and contrasts with other sections.

So I made two versions:
* with sections: https://deploy-preview-2413--determined-noyce-626d5b.netlify.com/user/reference/bionic/#python-support
* with a bulleted list: https://deploy-preview-2413--determined-noyce-626d5b.netlify.com/user/reference/bionic/#python-support

Since you have commit rights for the PR branch, you can choose either.